docs/report/Report.tex

```diff
@@ -0,0 +1,1074 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Looks great! The only bit I'd mention is the current placeholder report file is called `main.tex`, while this is called `Report.tex`. But I can sort that in a follow on PR if you want

Reply:
Thanks a lot Tiernan, if you could kindly amend the name change. I will check my work again for punctuation, grammar, omission or any errors and will reupload again with some references included.

Reply:
Should be done for you now, all you should need to do is run `git pull` on your system to update it on your side.